include/onnxruntime/core/platform/EigenNonBlockingThreadPool.h

```diff
@@ -431,6 +432,7 @@ class ThreadPoolTempl : public onnxruntime::concurrency::ExtendedThreadPoolInter
 
     worker_data_.resize(num_threads_);
     for (int i = 0; i < num_threads_; i++) {
+      worker_data_[i].set_denormal_as_zero = thread_options.set_denormal_as_zero;
```

Comment:
(Sorry had this comment earlier, but I don't see what happened to it): Can set_denormal_as_zero just be a flag at the same level as allow_spinning_ and not need to be in the worker data?

Reply:
Good point. Fixed.